functions/processQueueItem.js

```diff
@@ -2,16 +2,28 @@
 
 const AWS = require("aws-sdk");
 const S3 = new AWS.S3({ apiVersion: "2006-03-01" });
-const SQS = new AWS.SQS({ apiVersion: "2012-11-05" });
+const documentClient = new AWS.DynamoDB.DocumentClient();
 const request = require("request-promise-native");
+const { RATE_LIMIT, RATE_PERIOD, RATE_WAIT } = require("../lib/constants");
 const Metrics = require("../lib/metrics");
 
-module.exports.handler = async function(
-  { ReceiptHandle, Body },
-  { awsRequestId }
-) {
+exports.handler = async function({ Records }, context) {
+  console.log("Received", Records.length, "messages to process");
```

Comment:
This handler could receive as many as 5 items in a batch, but in practice I'm only seeing 1 item at a time roll in since these are invoked in parallel by the poller.